autotest/gcore/vrt_read.py

```diff
@@ -1454,6 +1454,9 @@ def test_vrt_protocol():
     ds = gdal.Open("vrt://data/byte_with_ovr.tif?ovr=0")
     assert ds.RasterXSize == 10
 
+    ds = gdal.Open("vrt://data/8bit_pal.bmp?expand=rgb")
```

Comment:
Move that to a new test method with a
``@pytest.mark.require_driver("BMP")``
decoration as in minimal builds the BMP driver isn't built

Reply:
done, ty